dataprofiler/labelers/utils.py

```diff
@@ -1,7 +1,12 @@
 """Contains functions for checking for installations/dependencies."""
 import sys
 import warnings
-from typing import Any, Callable, List
+from typing import Any, Callable, List, Union
+
+import numpy as np
+import pandas as pd
+
+DataArray = Union[pd.DataFrame, pd.Series, np.ndarray]
```

Comment:
If we are going to create our own. I might suggest we start following a similar pardigm to `numpy` and utilize our own `_typing` folder or `_typing.py` file at the root for universal typing that can be applied throughout the repo.

https://github.com/numpy/numpy/tree/main/numpy/_typing

Reply:
added `_typing.py`